CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -0,0 +1,73 @@
+# Changelog 1.12
+
+## 1.12.0 - 2023-05-24
+
+- The Multicast, TopologyAwareHints, NodeIPAM features are graduated from Alpha to Beta and therefore enabled by default.
+
+### Added
+
+- Add a new 'Failed' phase in ANP status for the case when all Agents have reported the status and at least one failure is received. ([#4608](https://github.com/antrea-io/antrea/pull/4608), [@wenyingd])
+- Add two new fields 'sourcePort' and 'sourceEndPort' in Antrea-native policy API to match traffic initiated from specific ports. ([#4687](https://github.com/antrea-io/antrea/pull/4687), [@Dyanngg])
+- Add a new field 'logLabel' to ACNP and ANP CRDs to specify firewall log label for Audit Logging. ([#4748](https://github.com/antrea-io/antrea/pull/4748), [@qiyueyao])
+- Add a new Controller configuration 'clientCAFile' to allow user to specify client CA. ([#4664](https://github.com/antrea-io/antrea/pull/4664), [@wenyingd])
+- Add support for ExternalIP in AntreaProxy to allow a Service to be accessed from outside the cluster using an external IP address. ([#4866](https://github.com/antrea-io/antrea/pull/4866), [@hongliangl])
+- Add OVS table name as label for 'ovs_flow_count' Prometheus metrics. ([#4893](https://github.com/antrea-io/antrea/pull/4893), [@cr7258])
+- Add WireGuard tunnel mode for Antrea Multi-cluster to support encryption of the traffic between member clusters. ([#4737](https://github.com/antrea-io/antrea/pull/4737) [#4606](https://github.com/antrea-io/antrea/pull/4606), [@hjiajing])
+- Add WireGuard encryption tunnel mode to Multi-cluster user guide. ([#4848](https://github.com/antrea-io/antrea/pull/4848), [@hjiajing])
+- Add support for EndpointSlice API for Multi-cluster Services. When the EndpointSlice API is available for the cluster, EndpointSlice resources of the exported Service, rather than the Endpoints resource, will be watched. For the Pod IP type of Services, EndpointSlice resources will be exported as the Multi-cluster Service's endpoints. ([#4895](https://github.com/antrea-io/antrea/pull/4895), [@luolanzone])
+- Add IP based group association query API and ClientSet for querying the Antrea Group/ClusterGroups. ([#4807](https://github.com/antrea-io/antrea/pull/4807), [@Dyanngg])
+- Add new exporter to FlowAggregator to write flows to local file. ([#4855](https://github.com/antrea-io/antrea/pull/4855), [@antoninbas])
+- Support new OS openEuler 22.03 and add it in tested distributions. ([#4957](https://github.com/antrea-io/antrea/pull/4957), [@ceclinux])
```

Comment:
"Add openEuler 22.03 as a new supported OS of Antrea, and update the [Kubernetes installer document](../docs/kubernetes-installers.md) with the information."

Reply:
Updated.